CONTRIBUTING.md

```diff
@@ -1,57 +1,108 @@
-# Contrubution guide
-This is a contribution guide for this repository. Recommend to read this before working on the project.
+# Contribution Guide
 
-# Branch Organization
+Welcome to the Shardeum contribution guide! This document provides the essential information needed to contribute effectively to our repository. Whether you're fixing a bug, adding a new feature, or improving documentation, following these guidelines will streamline the process and ensure smooth collaboration. Please read this guide carefully before contributing.
 
-- This repo has two long running branch, `main` and `dev`. 
-- The two branch are protected.
-- All new branch must be on top of or fork from developement branch `dev`.
-- All branches submiting to development branch must be tested.
+---
 
-This repo try to keep a clean single straight git history line.
+## Branch Organization
 
-# Pull Requests
+- **Branching Strategy:**
+  - The repository uses two primary branches:
+    - `main` for stable releases.
+    - `dev` for ongoing development.
+  - Both branches are **protected** to prevent direct commits. All changes must be introduced via pull requests.
```

Comment:
```suggestion
    - The dev branch is protected to prevent direct commits; all changes must be introduced via pull requests.
```
we are no longer using the main branch for development purposes, let's remove it 💪